visualization/app/codeCharta/util/nodeDecorator.ts

```diff
@@ -95,7 +95,7 @@ export class NodeDecorator {
 			let root = d3.hierarchy<CodeMapNode>(map)
 			root.leaves().forEach(node => {
 				metricData.forEach(metric => {
-					if (!node.data.attributes.hasOwnProperty(metric.name)) {
+					if (node.data.attributes[metric.name] === undefined) {
```

Comment:
```suggestion
					if (!node.data.attributes[metric.name]) {
```

Reply:
GIVEN `const metricValue =  0`

THEN `!metricValue` is `true` 

AND `metricValue === undefined` is `false`

Reply:
The only other solution is using 
```
Object.prototype.hasOwnProperty(node.data.attributes, metric.name)
```